pkg/cmd/infra/router/f5.go

```diff
@@ -76,6 +78,7 @@ type F5Router struct {
 
 // Bind binds F5Router arguments to flags
 func (o *F5Router) Bind(flag *pflag.FlagSet) {
+	flag.StringVar(&o.RouterName, "name", "public", "The name the router will identify itself with in the route status")
```

Comment:
The `oadm router [name]` command allows a router to be named. Could just pass it via an environment variable (e.g. `ROUTER_NAME`) and then use that here: `util.Env("ROUTER_NAME", "public")`.